homeassistant/components/sensor/broadlink.py

```diff
@@ -129,7 +129,7 @@ def _update(self, retry=3):
             if retry < 1:
                 _LOGGER.error(error)
                 return
-        except vol.Invalid:
+        except (vol.Invalid, vol.MultipleInvalid):
```

Comment:
`MultipleInvalid` is a child class of `Invalid`, so this change shouldn't be necessary. What behavior do you see?